src/views/Summary.js

```diff
@@ -0,0 +1,31 @@
+import AddExpensesForm from '../components/AddExpensesForm';
+
+const Summary = () => {
+  return (
+    <main className="container-md">
+      <header className="row text-center my-5">
+        <h1 className="display-1 text-primary">Mat's Budget</h1>
+      </header>
+      <section className="row my-4">
+        <div className="text-center">
+          <h2 className="display-2">Summary</h2>
+        </div>
+        <div className="">
+          <h3 className="fw-light">
+            Your monthly budget is <span className="text-primary">€7777</span>
```

Comment:
maybe already pass the dummy data as props, so we know what signature this component will have later.